AzureData/Source/ResourceCache.swift

```diff
@@ -34,15 +39,16 @@ public class ResourceCache {
 
     
     static var dispatchQueue: DispatchQueue { return DispatchQueue.global(qos: .default) }
-    
+
+    static var dataEncryptor: DataEncryptor? = nil
```

Comment:
We'll need to add a pass-through property to `AzureData.swift` to allow the user to set this `dataEncryptor` (or `resourceEncryptor`).  Something along the lines of:

```swift
public var resourceEncryptor: ResourceEncryptor {
    get { return ResourceCache.resourceEncryptor }
    set { ResourceCache.resourceEncryptor = newValue }
}
```